docs/user/unattended/storage.md

```diff
@@ -0,0 +1,956 @@
+# Storage configuration
+
+The general concepts regarding configuration of storage devices with Agama are exposed at the
+[corresponding subsection](../interactive/storage.md) of the interactive installation documentation.
+It is recommended to read that section before diving into the details below.
+
+:::warning Under development
+This document mentions some options that may still not be available in the current
+implementation of Agama.
+:::
+
+## Unattended installation using the legacy AutoYaST mode
+
+Most Agama profiles contain a `storage` section with the configuration settings that are detailed in
+subsequent sections of this document. But in some cases, `storage` can be replaced by a special
+section `legacyAutoyastStorage`, which is a 1:1 representation of the XML AutoYaST profile. This
+section supports everything offered by the `partitioning` AutoYaST section. Note that Agama does not
+validate this special section, so be careful to provide valid AutoYaST options.
+
+~~~json
+{
+  "legacyAutoyastStorage": [
+    {
+      "use": "all",
+      "partitions": []
+    }
+  ]
+}
+~~~
+
+Although that special section is offered for backwards compatibility and to ease gradual migration
+from AutoYaST to Agama, there are no plans to introduce any improvement or new feature in that
+legacy mode.
+
+## Basic structure of the Storage section
+
+As mentioned above, the Agama process to setup the storage devices can be configured using a
+`storage` section at the Agama configuration profile. A formal specification of the outer level of
+that section would look like this.
+
+```
+Storage
+  drives <Drive[]>
+  volumeGroups <VolumeGroup[]>
+  mdRaids <MdRaid[]>
+  btrfsRaids <BtrfsRaid[]>
+  nfsMounts <NFS[]>
+  boot [BootSettings]
+```
```

Comment:
I would avoid this "formal" specification in the users' documentation. Instead, I would use incomplete examples with references to the schema:

~~~
"storage": {
  "drives": [ ... ],
  "volumeGroups": [ ... ],
  "mdRaids": [ ... ]
}
~~~

Reply:
Done.